### DISCUSSION TITLE
Request to change the cache miss error handling in version 7.4.6.

### DISCUSSION THREAD

**Discussion by noobnoobnewbielife** [spatie/laravel-responsecache#430] [General]
I recently upgraded to laravel-responsecache version 7.4.6 and encountered an error related to cache miss. This is causing continuous error messages in the logging system and the situation where it cannot be deleted arbitrarily.

I think it would be better for users to handle this issue separately, just like in previous versions.

**Comment by swichers** [comment]
Possibly this #342
